src/main/java/net/earthcomputer/clientcommands/ClientCommands.java

```diff
@@ -22,6 +22,8 @@ public void onInitializeClient() {
         //noinspection ResultOfMethodCallIgnored
         configDir.mkdirs();
 
+        RenderQueue.add(RenderQueue.InjectLoc.HAND, ChorusManipulation::renderChorusGoal);
```

Comment:
What I was more thinking, is you add shapes from the tick loop, in amongst your other code. It's more convenient that way because you'll have access to all the temporary variables you're using. You add the shapes themselves to the render queue, and they'll then be rendered in the render loop.

Reply:
I'm not sure about that, since the way it's at the moment,  
if somebody decides to add something to the render queue in the future,
they can add their own render pipeline.
There they than can do whatever they want.
Here for the cglow command I had updated, the cglow had it's own renderFunction
https://github.com/Earthcomputer/clientcommands/pull/167/files#diff-49d93d00b6c6b62ab7e077685034fb2dd58e70fc76174acb59aea70e5152ed1aR134